### PR TITLE
Automate screenshotting for the readme WD-11363

### PR DESCRIPTION
**Pull request by edlerd**
## Done

- test to be run manually, which will create screenshots for the readme
- update screens in the readme

# Screenshots
create-instance
instance-graphical-console
instance-list
instance-terminal
operations-list
profile-list
storage-pool-list
warnings-list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,12 +34,12 @@ LXD-UI is a single page application written in TypeScript and React. See [Archit
...
 |-----------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
-| ![0create](https://github.com/canonical/lxd-ui/assets/1155472/7f0c45a6-2ba2-4cc7-bd7c-c0ebca76d648) | ![1instance-overview](https://github.com/canonical/lxd-ui/assets/1155472/c71d2153-ea71-4ecb-ab25-fabcd6fb1e55) | ![2instance-term](https://github.com/canonical/lxd-ui/assets/1155472/c2b741e2-8806-4d4d-9a9a-f536f76a13b9) |
+| ![0create](https://github.com/canonical/lxd-ui/assets/1155472/8c4f5eee-9d5a-40ca-93e1-57b1c393dbd9) | ![1instance-overview](https://github.com/canonical/lxd-ui/assets/1155472/af4a92ce-e562-43eb-945f-98b78b4bb03e) | ![2instance-term](https://github.com/canonical/lxd-ui/assets/1155472/14eaaffb-c770-4f34-936f-075ceb6be42e) |
```
Comment (mas-who) [inline README.md:37]: for my understanding, where do we get these uuids?

File: tests/helpers/instances.ts [hunk 2]
```diff
@@ -8,9 +8,11 @@ export const randomInstanceName = (): string => {
 export const createInstance = async (
   page: Page,
   instance: string,
-  type = "container",
+  type: string = "container",
+  project: string = "default",
+  image: string = "alpine/3.19/cloud",
 ) => {
-  await page.goto("/ui/");
+  await page.goto(`/ui/project/${project}`);
```
[the remark this answers is not in the document]
Comment (mas-who) [reply tests/helpers/instances.ts:15]: Also in test environment we are always using the default project right?

[the remark this answers is not in the document]
Comment (mas-who) [reply tests/helpers/instances.ts:15]: never mind, I see this is used in the screenshot tests :+1:

**Review by mas-who — APPROVED** [review]

**Comment by mas-who** [comment]
LGTM, thanks for automating this!

File: README.md (hunk 1 above)
Comment (edlerd) [reply README.md:37]: I added the screens to the description of this PR and then used the ids that github created in our readme.

**Pull request merged**